modules/node-certificates.adoc

```diff
@@ -0,0 +1,29 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="node-certificates_{context}"]
+= Node certificates
+
+== Purpose
+
+The Kubelet, in 4.x, uses the bootstrap certificate located in
+`/etc/kubernetes/kubeconfig` to initially bootstrap. This is followed by the
+link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#bootstrap-initialization[bootstrap
+initialization process] and
+link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#authorize-kubelet-to-create-csr[authoization
+of the kubelet to create a CSR].
+
+In that process, The kubelet generates a CSR while communicating over the
+bootstrap channel. The controller manager signs the CSR, resulting in a
+certificate  that the kubelet manages.
+
+== Expiration
+This bootstrap certificate is valid for 10 years.
+
+The kubelet-managed certificate is valid for 1 year and rotates automatically at
+around the 80 percent mark of that one year.
+
+== Customization
```

Comment:
Consider a section on renewing - linking to backup and restore docs. 
Consider a section on adding/removing nodes  (invalidating certificates) - liking to sections/content on that.